todo-app/src/App.js

```diff
@@ -1,13 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
+import Tasks from './Tasks'
+import action from './actions'
 
-class App extends Component {
+export default class App extends React.Component {
   render() {
     return (
       <div>
-        Hello World!
+        <h1>ToDo List:</h1>
+        <div>
+          {action.map((action, index) => {
+            return (
+              <Tasks
```

Comment:
This should probably refer to a single item as it accepts a single action object. For example `Task`?